pulsetranslator/pulsetranslator.py

```diff
@@ -25,6 +25,9 @@
 
 
 class PulseBuildbotTranslator(object):
+    # Available buildbot results
+    # http://hg.mozilla.org/build/buildbot/file/08b7c51d2962/master/buildbot/status/builder.py#l25
+    results = ['success', 'warnings', 'failure', 'skipped', 'exception', 'retry']
```

Comment:
I think we can leave it as you've written it for now.  Pulsetranslator will have to have some surgery regardless when we start running builds in TaskCluster.